config/config.exs

```diff
@@ -47,6 +47,17 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+config :lunchbot, Lunchbot.Scheduler,
+  jobs: [
+    # Bldr reminders at noon Monday and Thursday
+    {"0 18 * * 1,4",
```

Comment:
might be a better pattern to run this job every hour and check which offices we can run this query for.